_invoke

```diff
@@ -3,25 +3,40 @@
 typeset -A opt_args
 local context state line
 
-_invoke_subcommand () {
-  if [ -f tasks.py ]; then
-    local cache_policy
+function _invoke_subcommands {
+    COMMAND_COMPLETIONS=( $(invoke -l |tr -d ' ' | tail -n+2 | sed '$ d' | tr '\n' ' ') )
```

Comment:
Ahh I see. Yeah, I didn't realize that docstrings were included in the output of -l. will do